pkg/secretless/config/v2/config.go

```diff
@@ -78,7 +78,20 @@ func NewConfigsByType(
 		rawConfigs = append(rawConfigs, *cfg)
 	}
 
-	httpConfigs, tcpConfigs := separatedHTTPAndTCPConfigs(rawConfigs, availPlugins)
+	var httpConfigs , tcpConfigs, sshConfigs []Service
+
+	for _, cfg := range rawConfigs {
+		if sharedobj.IsHTTPPlugin(availPlugins, cfg.Connector) {
+			httpConfigs = append(httpConfigs, cfg)
+			continue
+		} else if cfg.Connector == "ssh" {
+			sshConfigs = append(sshConfigs, cfg)
+			continue
+		}
```

Comment:
unrelated/nit/notablocker: we might want this giant select generalized if we add one more somehow